README.md

```diff
@@ -124,7 +124,7 @@ set -g theme_newline_prompt '$ '
 **Prompt options**
 
 - `theme_display_ruby`. Use `no` to completely hide all information about Ruby version. By default Ruby version displayed if there is the difference from default settings.
-- `theme_display_node`. If set to `yes`, will display current NVM or FNM node version.
+- `theme_display_node`. If set to `yes`, will display current NVM or FNM node version. If set to `rc`, will display the version if an `.nvmrc` or `.node-version` file is found. If set to `package`, will display the version if a `package.json` file is found.
```

Comment:
Fine by me, but it'd be a breaking change with the old behavior which I was trying to avoid. I'll do it in a separate commit so it's easy to revert this one comment if you want.